### DISCUSSION TITLE
PLEASE BE CAREFUL

### DISCUSSION THREAD

**Discussion by LiaoxyuCM** [community/community#132063] [Other Feature Feedback, Questions, & Ideas]
### Select Topic Area

Question

### Body

Csdn.net have a new Code hosting website Gitcode.com

Gitcode **without our consent copy ours repository intact**

(I think shouldn't is Gitcode should is Gitcopy)

**Comment by LiaoxyuCM** [comment]
and Gitcode automatically **create an account** after copying the code.

**Reply by LiaoxyuCM** [reply]
Do you think you have management rights when you log in to your gitcode account with github authorization?
No!
GitCode need you agree  message:”_GitCode read your github’s privacy information_”
Only then can you delete your repos

**Comment by LiteBrite82** [comment]
Thank you kindly for reaching out. If you are experiencing behavior against our Terms of Service or need to report GitHub account-related abuse, you can reach out to our private support team. We are unable to assist with or discuss personal account details here in Discussions.

Please visit this link for more details: https://docs.github.com/en/communities/maintaining-your-safety-on-github/reporting-abuse-or-spam We hope that helps!
